partiql-planner/src/main/kotlin/org/partiql/planner/internal/transforms/RelConverter.kt

```diff
@@ -437,13 +437,17 @@ internal object RelConverter {
          *         1. Ast.Expr.SFW has every Ast.Expr.CallAgg replaced by a synthetic Ast.Expr.Var
          *         2. Rel which has the appropriate Rex.Agg calls and groups
          */
-        private fun convertAgg(input: Rel, select: QueryBody.SFW, groupBy: GroupBy?, env: Env): Pair<QueryBody.SFW, Rel> {
```

Comment:
nit: for all of the aggregation rewriting, I think we should move `convertAgg` and stuff related to `AggregationTransform` into another file since this file's getting pretty large.